README.md

```diff
@@ -37,25 +61,63 @@ Last, redbiom itself can be installed as a normal Python package.
     $ git clone https://github.com/wasade/redbiom.git
     $ pip install -e .    
 
+# Terminology and notation
+
+In redbiom, the word "context" refers to a way in which the sample data were processed. Data are loaded into contexts and searches for samples by observation happen within contexts.
+
+To support the one to many relationship between a sample's metadata and its data, within a context, a sample's IDs are prefixed by a "tag" which can be specified at load. Internally, within a context, these IDs are of the form `<tag>_<sample-id>`. The use of the `_` character ensures that they are not valid QIIME sample IDs, and is necessary so we can appropriately differentiate these IDs. Methods which write data will coerce these invalid QIIME IDs into valid IDs of the form `<sample-id>.<tag>`. **IMPORTANT**: if you run your own resource, it is important to specify `--tag` on load of sample data to differentiate BIOM tables in which the sample IDs between the tables may not be mutually exclusive. 
```

Comment:
It's super important if you do not have unique sample IDs, which is not common at this time in the field AFAIK